### PR TITLE
fix(test): unignore __stories__ from babel. Fixes #862

### LINKED ISSUE #862: "[BUG]" Venia Storybook Crash
**Describe the bug**
Running `yarn storybook:venia` results in the Storybook "red screen of death" with the error message:

> Module parse failed: Unexpected token (15:8) You may need an appropriate loader to handle this file type. | | return ( > <div style={styles}> | <BraintreeDropin onError={() => {}} onSuccess={() => {}} /> | </div>

**To Reproduce**
1. Checkout the latest of `release/2.0` branch. As of this writing it is [this commit](https://github.com/magento-research/pwa-studio/commit/625dd5130dd318f5c3166f97b6305ca028085638)
1. `yarn clean:all`
1. `yarn install`
1. `yarn build`
1. `yarn storybook:venia`

Observe the red screen of death.

**Expected behavior**
Storybook loads and behaves as expected.

**Screenshots**

**Please complete the following device information:**
 - Device: MacBook Pro
 - OS: macOS Mojave 10.14.2
 - Browser: Chrome
 - Version: 71
 - Node Version `node -v`: 10.15.0

**Please let us know what packages this bug is in regards to:**
- [x] `venia-concept`
- [ ] `pwa-buildpack`
- [ ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

**Possible solutions**

This likely has to do with some last minute additions to the Braintree PR that lazy loaded the `BraintreeDropin` component.

This may also have to do with a recent upgrade to React `16.8`.

### PR DESCRIPTION
**Pull request by zetlen**
## Description
The root babel.config.js had a generic ignore statement that wouldn't compile or copy anything from a folder with double underscore delimiters, like `__tests__`. But Storybook wants Babel to compile `__stories__`, so this PR adjusts the ignore statement to allow `__stories__` to be compiled.

## Related Issue

Closes #862.

## Motivation and Context
Storybook was broken and it needs to work.

## How Has This Been Tested?

Tested locally.

## Proposed Labels for Change Type/Package

BUG, TEST in peregrine and venia-concept

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have linked an issue to this PR.
- [x] I have indicated the change type and relevant package(s).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] All CI checks are green (linting, build/deploy, etc).
- [ ] At least one core contributor has approved this PR.

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ const config = api => {
          * package root, like 'src/classify'.
          */
         development: {
-            // Ignore __mocks__, __stories__, etc.
-            ignore: [/\/__\w+__\//],
```
Comment (tjwiebell) [inline babel.config.js]: This same pattern is in `packages/pwa-buildpack/.babelrc.js`. I don't see a use case for pwa-buildpack to have any ui components, but if you wanted to future proof that package, change it there too.

**Comment by tjwiebell** [comment]
Tracked down fix for Kebab, icon names need updated.

`packages/venia-concept/src/components/MiniCart/__stories__/Kebab.js`
```js
stories.add('Kebab Open', () => (
    <div style={styles}>
        <Kebab isOpen={true}>
            <Section icon="Heart" text="Section 1" />
            <Section icon="Edit2" text="Section 2" />
            <Section icon="Trash" text="Section 3" />
            <Section text="Non-icon Section" />
        </Kebab>
    </div>
));
```

**Comment by zetlen** [comment]
Very good catch on all this stuff, @tjwiebell . Made the changes and pushing.

File: packages/venia-concept/src/components/MiniCart/kebab.js [hunk 2]
```diff
@@ -19,6 +19,9 @@ class Kebab extends Component {
     componentDidMount() {
         document.addEventListener('click', this.handleDocumentClick);
         document.addEventListener('touchend', this.handleDocumentClick);
+        if (this.props.isOpen) {
+            this.setState({ isOpen: true });
+        }
```
Comment (jimbo) [inline packages/venia-concept/src/components/MiniCart/kebab.js]: Generally, initial state that depends on props [should be set in the constructor](https://reactjs.org/docs/react-component.html#constructor).

```es6
constructor(props) {
  super(props)

  this.state = { 
    isOpen: !!props.isOpen
  }
}
```

File: packages/venia-concept/src/components/MiniCart/__stories__/Kebab.js [hunk 3]
```diff
@@ -22,9 +22,9 @@ stories.add('Kebab Closed', () => (
 stories.add('Kebab Open', () => (
     <div style={styles}>
         <Kebab isOpen={true}>
-            <Section icon="heart" text="Section 1" />
-            <Section icon="x" text="Section 2" />
-            <Section icon="chevron-up" text="Section 3" />
+            <Section icon="Heart" text="Section 1" />
```
Comment (sirugh) [inline packages/venia-concept/src/components/MiniCart/__stories__/Kebab.js]: Can you add an enum to the propTypes of `MiniCart/Kebab`:

```
propTypes: {
  icon: oneOf(['Heart',...])
}
```

**Pull request merged**